packages/block-library/src/navigation/style.scss

```diff
@@ -290,18 +290,25 @@
 }
 
 // Default background and font color.
-.wp-block-navigation:not(.has-background) {
```

Comment:
I can't reproduce the issue with `has-background` not being applied on the front end. I'm seeing whether the background color is custom, theme defined or a default core one. Which theme are you using? 

Regardless, the change is working well on the front end. But in the editor, I'm seeing this weird border at the bottom of each dropdown when nested 2 or more deep:

<img width="609" alt="Screen Shot 2021-11-10 at 4 20 35 pm" src="https://user-images.githubusercontent.com/8096000/141054746-004be504-50f8-46c1-9d55-a0f626eacb5a.png">

I think it's happening because the block appender doesn't get `background-color` set inline, like the navigation items do, so the box-shadow becomes visible.

Reply:
Thank you for testing. It does look like those borders need a little more work, I'll dive in when I get my dev environment working again.

In the mean time, to clarify: `has-background` is applied correctly when the `Background` color is set. But it is not applied when _only_ the `Submenu & overlay background` color is set. I don't even think it should be, at least not on the main navigation block, possibly on submenus.